src/deluge/storage/flash_storage.cpp

```diff
@@ -636,6 +641,7 @@ void readSettings() {
 		gui::menu_item::fillColourMenu.value = gui::menu_item::Colour::AMBER;
 		gui::menu_item::onceColourMenu.value = gui::menu_item::Colour::MAGENTA;
 	}
+	defaultSliceMode = static_cast<SampleRepeatMode>(buffer[167]);
```

Comment:
Similar to the checks above, we should make sure the value at `buffer[167]` is within the valid range for `SampleRepeatMode`.